docs/self-managed/platform-deployment/kubernetes-helm.md

```diff
@@ -137,3 +137,73 @@ If the output of the `describe` command was not beneficial, tail the logs of the
 ```
 > kubectl logs -f <POD NAME> 
 ```
+
+## Upgrading from one Helm release to another
+
+To upgrade to a more recent version of the Camunda Platform Helm charts, there are certain things you need to keep in mind.
+
+Normally for a Helm upgrade, you run the
+[Helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) command. If you have disabled Identity and the related authentication mechanism, you should be able to easily do an upgrade via this command, `helm upgrade`.
+
+However, if Identity is enabled (this is the default), the upgrade path is a bit more complex than just running `helm upgrade`.
+
+Read the next sections to familiarize yourself with the upgrade process.
+
+### Upgrading with Identity and Secrets
+
+If you have installed the Camunda Platform 8 Helm charts before with default values, this means Identity and the related authentication mechanism are enabled. For authentication, the Helm charts generate for each web app the secrets randomly if not specified on installation. If you just run `helm upgrade` to upgrade to a newer chart version, you likely will see the following return:
+
+```shell
+$ helm upgrade camunda-platform-test camunda/camunda-platform 
+Error: UPGRADE FAILED: execution error at (camunda-platform/charts/identity/templates/tasklist-secret.yaml:10:22): 
+PASSWORDS ERROR: You must provide your current passwords when upgrading the release.
+                 Note that even after reinstallation, old credentials may be needed as they may be kept in persistent volume claims.
+                 Further information can be obtained at https://docs.bitnami.com/general/how-to/troubleshoot-helm-chart-issues/#credential-errors-while-upgrading-chart-releases
+
+    'global.identity.auth.tasklist.existingSecret' must not be empty, please add '--set global.identity.auth.tasklist.existingSecret=$TASKLIST_SECRET' to the command. To get the current value:
+
+        export TASKLIST_SECRET=$(kubectl get secret --namespace "zell-c8-optimize" "camunda-platform-test-tasklist-identity-secret" -o jsonpath="{.data.tasklist-secret}" | base64 --decode)
+```
+
+As described earlier per default, this occurs because Secrets are randomly generated on the first Helm installation if not further specified. We use a library chart [provided by Bitnami](https://github.com/bitnami/charts/tree/master/bitnami/common) for this. The generated Secrets are persisted on persistent volume claims (PVCs), which are not maintained by Helm.
```

Comment:
@christinaausley somehow this sounds for me not round. Does it make sense to remove the `per default` ? :thinking: But might also be because I'm not a native speaker, so I'm not sure.